bullet-train.zsh-theme

```diff
@@ -13,6 +13,9 @@
 # The default configuration, that can be overwrite in your .zshrc file
 # ------------------------------------------------------------------------------
 
+# space variable to make spaces more visible and avoid bugs
+space=' '
```

Comment:
Hmm, this variable will be visible outside the script? Is it wise to use such generic name for it?